README.md

```diff
@@ -624,6 +624,20 @@ For example, to specify `hci1`:
 sudo NOBLE_HCI_DEVICE_ID=1 node <your file>.js
 ```
 
+If you are using multiple HCI devices in one setup you can run two instances of noble with different binding configurations by initializing them seperatly in code:
+
+```
+const HCIBindings = require('noble/lib/hci-socket/bindings');
```

Comment:
shouldnt it align to  require('@abandonware/noble'); ?

or otherwise please add:
 npm install bleno@npm:@abandonware/bleno 
and remote @abanonware prefix elsewhere